Sources/NIOExtras/LengthFieldBasedFrameDecoder.swift

```diff
@@ -13,6 +13,38 @@
 //===----------------------------------------------------------------------===//
 
 import NIO
+import Foundation
+
+extension ByteBuffer {
+    @inlinable
+    mutating func get24UInt(
+        endianness: Endianness = .big
```

Comment:
`get*` functions typically take an index, so I'd suggest you add `index: Int` here and use that instead of the `readerIndex` below to be more in line with the other `get*` functions.

Reply:
Changed :)